GliaWidgets/StaticValues.swift

```diff
@@ -6,5 +6,5 @@ final class StaticValues {
     /// version cannot be changed by integrators, so this ensures that our code will
     /// always have the correct version regardless of what our integrators do with
     /// our plist files.
-    static let sdkVersion = "0.10.11"
+    static let sdkVersion = "2.0.0"
```

Comment:
There will be conflicts between this PR and [this change](https://github.com/salemove/ios-sdk-widgets/pull/620/files#diff-734869ca4d1ab41db1510b505ab5c60eca8684001b16ea08d5da34d203cfe8e5R9) made in that PR